easybuild/framework/easyconfig/style.py

```diff
@@ -68,6 +79,104 @@
 # https://pycodestyle.readthedocs.io or more specifically:
 # https://pycodestyle.readthedocs.io/en/latest/developer.html#contribute
 
+
+def _check_param_group(params, done_params):
+    """
+    Check whether the specified group of parameters conforms to the style guide w.r.t. order & grouping of parameters
+    """
+    result, fail_msgs = None, []
+
+    fail_msgs.extend(_check_param_group_head(params, done_params))
+
+    if fail_msgs:
+        result = (0, "W001 %s" % ', '.join(fail_msgs))
```

Comment:
Yes, see https://pycodestyle.readthedocs.io/en/latest/intro.html#error-codes, but I'm open to suggestions for a better choice...